debug_toolbar/templates/debug_toolbar/panels/sql.html

```diff
@@ -56,7 +56,9 @@
 						{% endif %}
 					</td>
 					<td class="djdt-timeline">
-						<div class="djDebugTimeline"><div class="djDebugLineChart{% if query.is_slow %} djDebugLineChartWarning{% endif %}" style="left:{{ query.start_offset|unlocalize }}%"><strong style="width:{{ query.width_ratio_relative|unlocalize }}%;background-color:{{ query.trace_color }}">{{ query.width_ratio }}%</strong></div></div>
+						<svg class="djDebugLineChart" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 100 5" preserveAspectRatio="none" aria-label="{{ query.width_ratio }}%">
```

Comment:
This no longer contains `{% if query.is_slow %} djDebugLineChartWarning{% endif %}`. Is that intentional?

Reply:
I somehow missed that. Restored in 4965bf5.